lib/ansible/modules/identity/ipa/ipa_subca.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Abhijeet Kasurde (akasurde@redhat.com)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
I think we want to make this a one-liner now, as mentioned [here](https://github.com/ansible/community/wiki/Testing:-boilerplate,-wildcard-imports,-and-get_exception#boilerplate).

Reply:
This was written way back in time. Anyways I will add this.